locales/en-US/strings.properties

```diff
@@ -205,3 +205,6 @@ firstrun_privacy_notice=Privacy Notice
 
 firstrun_continue_to_login=Continue
 firstrun_skip_login=Skip this step
+
+# Tooltip for context menu
```

Comment:
This comment should have the format `#LOCALIZATION NOTE (context_menu_title):` so that the localisers know this comment is for them